packages/augur-core/source/libraries/ContractCompiler.ts

```diff
@@ -235,7 +237,13 @@ export class ContractCompiler {
                 remappings: [ `ROOT=${this.configuration.contractSourceRoot}/`],
                 optimizer: {
                     enabled: true,
-                    runs: 500
+                    runs: 200,
```

Comment:
Why only 200 runs here?  Feels like you would want 1000s of runs.
Note: "runs" means "number of times the contract is expected to be executed".  By default, Solidity optimizes for deployment size, more runs prefers optimizations that increase deployment size in favor of decreasing runtime costs.